engine/lib/access.php

```diff
@@ -93,14 +92,13 @@ function get_access_list($user_guid = 0, $site_guid = 0, $flush = false) {
  * @see get_write_access_array() for the access levels that a user can write to.
  *
  * @param int  $user_guid User ID; defaults to currently logged in user
- * @param int  $site_guid Site ID; defaults to current site
  * @param bool $flush     If set to true, will refresh the access ids from the
  *                        database rather than using this function's cache.
  *
  * @return array An array of access collections ids
  */
-function get_access_array($user_guid = 0, $site_guid = 0, $flush = false) {
-	return _elgg_services()->accessCollections->getAccessArray($user_guid, $site_guid, $flush);
+function get_access_array($user_guid = 0, $flush = false) {
+	return _elgg_services()->accessCollections->getAccessArray($user_guid, $flush);
```

Comment:
Keep the signature, change in access collections. We can then deprecated these and add an elgg_ version with the new signature